lib/Yars/Routes.pm

```diff
@@ -82,6 +82,13 @@ sub _get {
           || _redirect_to_remote_stash( $c, $filename, $md5 )
           || $c->reply->not_found;
     };
+
+    if ($c->req->headers->header('X-Use-X-Accel'))
+    {
+        $c->res->headers->add('X-Accel-Redirect', "$dir/$filename");
```

Comment:
Instead of "$dir/$filename" can we do "/static$dir/$filename", to avoid collisions between real files and yars routes?